traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/dns/ZoneManager.java

```diff
@@ -372,6 +411,51 @@ private static void generateZones(final TrafficRouter tr, final LoadingCache<Zon
 		}
 	}
 
+	private static void generateNewZones(final TrafficRouter tr,
+	                                     final SnapshotEventsProcessor snep, final LoadingCache<ZoneKey, Zone> zc,
+	                                   final LoadingCache<ZoneKey, Zone> dzc, final ExecutorService initExecutor) throws IOException {
```

Comment:
There's some mixing of tabs/spaces in the indentation in this file.

Reply:
done